pkg/ddl/ingest/backend.go

```diff
@@ -218,22 +227,6 @@ func (bc *litBackendCtx) Flush(ctx context.Context, mode FlushMode) (flushed, im
 		}
 	}
 
-	var newTS uint64
-	if mgr := bc.GetCheckpointManager(); mgr != nil {
-		// for local disk case, we need to refresh TS because duplicate detection
```

Comment:
For a subtask, we should use the same TS. But if disk is full and trigger a partitial import, we should switch TS. Because there maybe duplicate key before and after the  partitial import, and we need the different TS to let duplicate detection report error.

In current architecture we can't solve both problem, and because the team didn't arrange a plan for us to solve it. I prefer we keep this refreshTS logic. False alarm of duplicate enrty error is better than false negative that causes data-index inconsistency

Reply:
For now, the subtask executor can't update the subtask meta directly. Can we create another goroutine to update the subtask meta and ingest TS on demand, like `updateSubtaskSummaryLoop`? @lance6716 @D3Hunter

Reply:
I think we can keep current architecture and only improve a bit to mitigate the problem. We only solve this problem: if no paritial import happens, subtask worker failover should use the same TS. And the fix is, for the first (partial or full) import, use the TS saved in subtask metadata. For subsequential partitial import, acquire a new TS. Leave the issue that failover on subsequential partitial import will cause false alarm of "duplicate entry" error.

Reply:
another writer would make it complicate to manage and sync `meta` field, right now `OnFinished` is the only place meta is allowed to be changed.

if we gone to do it, maybe save this info in `summary`, then you can reuse `updateSubtaskSummaryLoop`